pychromecast/config.py

```diff
@@ -10,7 +10,7 @@
 APP_MEDIA_RECEIVER = "CC1AD845"
 APP_PLEX = "06ee44ee-e7e3-4249-83b6-f5d0b6f07f34_1"
 APP_DASHCAST = "84912283"
-APP_HOME_ASSISTANT = "B12CE3CA"
+APP_HOME_ASSISTANT = "A078F6B0"
```

Comment:
```suggestion
CAST_APP_ID_HOMEASSISTANT_LOVELACE = "A078F6B0"
```

Need to update pychromecast/controllers/homeassistant.py too